src/shape/bounding-box.js

```diff
@@ -73,6 +73,10 @@ pc.extend(pc, function () {
             this.type = src.type;
         },
 
+        clone: function () {
+            return new pc.BoundingBox(this.center, this.halfExtents);
```

Comment:
Those vectors are not cloned inside of a constructor, so by "cloning" you will have same vectors used by different bounding boxes, which will of course lead to wrong behaviour when changing one of clones or original properties.